distribution/debian/postrm

```diff
@@ -1,9 +1,36 @@
 #!/bin/sh
 set -e
 
-# In case this system is running systemd, we make systemd reload the unit files
-# to pick up changes.
-if [ -d /run/systemd/system ] ; then
-	systemctl --system daemon-reload >/dev/null || true
+BUILD_VERSION={version}
+UPDATER={updater}
+
+if [ $1 = "abort-install" ]; then
+  # preinst was aborted, possibly due to NzbDrone still running.
+  # Nothing to do here
+  :
+fi
+
+# The bin directory is expected to be empty, unless the BuiltIn updater added files.
+if [ $1 = "remove" ] && [ -d /usr/lib/sonarr/bin ]; then
+  rm -rf /usr/lib/sonarr/bin
+fi
+
+#BEGIN BUILTIN UPDATER
+# Remove any existing patch if still present
+if [ $1 = "remove" ] && [ -d /usr/lib/sonarr/bin_patch ]; then
+  rm -rf /usr/lib/sonarr/bin_patch
+fi
+#END BUILTIN UPDATER
+
+# Purge the entire sonarr configuration directory.
+# TODO: Maybe move a minimal backup to tmp?
```

Comment:
Synology doesn't use dpkg though. So that's gotta be handled specifically for their port. But I considered that we might just tar.gz the sonarr.db* files and dump that in temp. On the other hand, ppl have to explicitly call 'purge', so I was a bit torn on it.

Reply:
Yeah, different package, just mentioning the pain it can cause, though if you call `purge` you probably mean it, so it's not a big deal I feel.